src/visip/code/representer.py

```diff
@@ -27,6 +28,9 @@ def type_code(self, type_hint):
         :param type_hint:
         :return:
         """
+        return self.type_code_inner(dtype.to_typing(type_hint))
```

Comment:
This is a bit questionable. In order to keep original types, we should possibly keep a flag in the type types whether the conversion was performed.

Reply:
As the visip code should use dtype only, we should always convert typing to type in represented code.

Reply:
Will be implemented after merge.

Reply:
Make an issue for this.